ui-tests/__test__/flows/navigate-prompts/navigate-back-to-current-prompt-with-code-attachment.ts

```diff
@@ -0,0 +1,40 @@
+import { Page } from 'playwright/test';
+import { getSelector, waitForAnimationEnd } from '../../helpers';
+import testIds from '../../../../src/helper/test-ids';
+import { closeTab } from '../close-tab';
+import { openNewTab } from '../open-new-tab';
+
+export const navigateBackToCurrentPromptWithCodeAttachment = async (page: Page, skipScreenshots?: boolean): Promise<void> => {
+  await closeTab(page, false, true);
+  await openNewTab(page, false, true);
+
+  await page.locator(`${getSelector(testIds.prompt.input)}`).fill('This is the first user prompt');
```

Comment:
nit: redundant backticks

Reply:
Good nit, we have them quite a bit throughout the E2E tests. Will try to clean them up in my upcoming PR.